composer/modules/scripts/npm-postinstall.js

```diff
@@ -0,0 +1,19 @@
+const { log } = console;
+const path = require('path');
+const fs = require('fs');
+
+const cwd = process.cwd();
+
+const readablePl = fs.createReadStream(path.join(cwd, 'package-lock.json.bak'));
```

Comment:
Shall we add package-lock.json.bak to git ignore in case someone terminate the build.

Reply:
.bak files are already gitignored. https://github.com/ballerina-lang/ballerina/blob/master/composer/.gitignore#L26